API.md

```diff
@@ -110,10 +113,25 @@ This class is the wrapper for simplified models and the rest of the context need
 <a name="CommonModel"></a>
 
 ## CommonModel ⇐ [<code>CommonSchema&lt;CommonModel&gt;</code>](#CommonModel)
-Common representation for the renderers.
+Common internal representation for a model.
 
 **Kind**: global class  
 **Extends**: [<code>CommonSchema&lt;CommonModel&gt;</code>](#CommonModel)  
+**Properties**
+
+| Name | Type | Description |
+| --- | --- | --- |
+| $id | <code>string</code> | define the id/name of the model. |
+| type | <code>string</code> \| <code>Array.&lt;string&gt;</code> | this is the different types for the model. All types from JSON Schema are used with no custom ones added. |
```

Comment:
Is there any reason to use html tags instead of markdown here? E.g., you can achieve the same result in a more readable and simpler way:

```suggestion
| type | `string` \| `Array.<string>` | this is the different types for the model. All types from JSON Schema are used with no custom ones added. |
```

Reply:
This is generated by the command `npm run docs` so I have no control over the API, I just generated it manually for you to see the changes it would do. Normally the CI system auto-generates this when PRs are merged

Reply:
Oh didn't know about it 👍  thks